domains/zaidi.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "zaidi-01",
+    "email": "26808641+zaidi-01@users.noreply.github.com"
```

Comment:
can you replace email with either your email or leave it blank and add other socials? (like discord, twitter etc.)